templates/patterns/field_list/field_list.ui_patterns.yml

```diff
@@ -1,15 +1,22 @@
 field_list:
   label: "Field list"
   description: "A list of field components to display structured information."
+  variants:
+    default:
+      label: "Default"
+      description: "The default component modifier is vertical. It will align the field items vertically."
+    horizontal:
+      label: "Horizontal"
+      description: "It will align the field items horizontally."
   fields:
     items:
       type: "array"
       label: "Field items"
-      description: "List of field items, each having a 'label' and a 'body'"
+      description: "List of field items, each having a 'term' and a 'definition'"
       preview:
-        - label: "When"
-          body: "Thursday 15 November, 08:00 AM to Friday 16 November"
-        - label: "Where"
-          body: "The EGG, Rue Barra 175, 1070 Brussels, Belgium"
-        - label: "Live streaming"
-          body: "08:00 CET | 00d:23h:35m"
+        - term: "When"
+          definition: "Thursday 15 November, 08:00 AM to Friday 16 November"
+        - term: "Where"
+          definition: "The EGG, Rue Barra 175, 1070 Brussels, Belgium"
+        - term: "Live streaming"
+          definition: "08:00 CET | 00d:23h:35m"
```

Comment:
We don't change our field properties names to follow what ECL requires, we instead remap them in the Twig template itself (not in the preprocess as it would be too early).

Reply:
I think in this case its fine since this is a pure 2.x pattern so the names aren't set on stone yet.

Reply:
Yeah we could do that if they would make sense in our use case but also `term` / `definition` are just confusing.

Reply:
For @nagyad we should have something like this in the twig template (with the correct naming of course):

```twig
{% set _items = [] %}
{% for _item in items %}
  {% set _items = _items|merge([{
    label: _item.label,
    path: _item.href
  }]) %}
{% endfor %}
```